### PR TITLE
Add Vue-Storefront

### PR DESCRIPTION
**Pull request by bloodf**
Project URL: https://www.vuestorefront.io/

Vue Storefront 2 Example URL:

https://www.vans.ca/en-ca
https://puredailycare.com/

Vue Storefront 1 Example URL:

https://zadig-et-voltaire.com/

### CODE REVIEW

File: src/technologies.json [hunk 1]
```diff
@@ -23647,6 +23647,33 @@
       ],
       "website": "https://vuejs.org"
     },
+    "Vue-Storefront 1": {
+      "cats": [
+        6,
+        12
+      ],
+      "description": "Vue Storefront is the world's fastest-growing open-source eCommerce frontend.",
```
Comment (AliasIO) [inline src/technologies.json:23655]: Please change this to something more neutral, e.g. "Vue Storefront is a frontend platform for headless commerce."

**Comment by AliasIO** [comment]
Thanks for the PR! Could we combine the two into a single technology and capture the version number as `1` or `2`?

E.g. `"html": "<[^>]+\\sdata-v(?:ue)?-\\;version=2"`.

**Comment by bloodf** [comment]
> Thanks for the PR! Could we combine the two into a single technology and capture the version number as `1` or `2`?
> 
> E.g. `"html": "<[^>]+\\sdata-v(?:ue)?-\\;version=2"`.

The main problem is that Version 1, implies only Vue.js, and Version 2, implies Nuxt. That's why I didn't made just a single version :(

File: src/technologies.json (hunk 1 above)
Comment (bloodf) [reply src/technologies.json]: I've updated

**Comment by AliasIO** [comment]
> The main problem is that Version 1, implies only Vue.js, and Version 2, implies Nuxt. That's why I didn't made just a single version :(

What if you leave off the Nuxt.js implies? I think in most cases we identify Nuxt.js anyway.

**Comment by bloodf** [comment]
@AliasIO fixed the versioning detection

**Comment by AliasIO** [comment]
Perfect, thanks!

**Pull request merged**